app/controllers/admin/collections_controller.rb

```diff
@@ -347,7 +347,13 @@ def update_access_settings(collection, params)
       collection.default_hidden = params[:hidden] == "1"
     end
     if params[:save_field] == "cdl"
-      collection.cdl_enabled = params[:cdl] == "1"
+      case params[:cdl]
+      when "1"
+        collection.cdl_enabled = true
+      else
+        BulkActionJobs::ReturnCheckouts.perform_later(collection.id, nil)
```

Comment:
This is probably fine for now, but I'm wondering about hooking this in to a Admin::Collection callback (like how `around_save :reindex_members` works) instead of enqueuing it here.  This might have the benefit of working whether the collection gets CDL disabled via the UI, script, or rails console.  It could also move it later so that it only gets enqueued if the collection save is successful.  What do you think?